README.rdoc

```diff
@@ -48,6 +48,10 @@ For initial development, the plugin targets all communication at a vCenter insta
     knife[:vsphere_pass] = "your password"
     knife[:vsphere_dc] = "your-datacenter"
 
+The vSphere password can also be stored in a base64 encoded version (to visually obfuscate it) by prepending 'base64:' to your encoded password. For example:
+
+    knife[:vsphere_pass] = 'base64:VGhhbmtzRXpyYSE=\n'
```

Comment:
I'm not sure about that `\n`. It's going to come out as a literal `\n` and not a newline because of the single quoting. I think it's working for you because Ruby drops anything after the padding:

```
>> puts Base64.decode64 'VGhhbmtzRXpyYSE=\n'
ThanksEzra!
=> nil
>> puts Base64.decode64 'VGhhbmtzRXpyYSE=\nIamIgnored'
ThanksEzra!
=> nil
>> puts Base64.decode64 'VGhhbmtzRXpyYSE\nIamNotIgnored'
ThanksEzra!'!���� ���
=> nil
```